js/form.js

```diff
@@ -269,12 +269,20 @@ function setupBirthday(limits) {
 function isValid(element, value) {
   switch (element) {
     case "nickname":
+      let numberSpecialCharacters = 0
+      let nameWithoutSpecialCharacters = ''
+
+      for (let i = 0; i < value.length; i++) {
+        if (/[^a-zA-Z0-9\u00c0-\u017e\u0400-\u04FF ]/.test(value[i])) numberSpecialCharacters++
```

Comment:
It might make the check at the end more readable if you do something like
if (...) 
  numberSpecialCharacters++ 
elseif (... === " ") 
  numberSpaces++ 
else 
  numberRegularCharacters++